app/lib/common/toolkit/html_editor/html_editor.dart

```diff
@@ -578,49 +582,65 @@ class HtmlEditorState extends State<HtmlEditor> {
       return before;
     }
 
-    final roomId = widget.roomId;
-    // Inline Mentions
-    MentionAttributes? mention;
-    try {
-      mention =
-          attributes.entries
-                  .firstWhereOrNull((e) => e.value is MentionAttributes)
-                  ?.value
-              as MentionAttributes?;
-    } catch (e) {
-      // If any error occurs while processing mention attributes,
-      // fallback to default decoration
-      return defaultTextSpanDecoratorForAttribute(
-        context,
-        node,
-        index,
-        text,
-        before,
-        after,
-      );
-    }
-
-    if (mention != null && roomId != null) {
-      return WidgetSpan(
-        alignment: PlaceholderAlignment.middle,
-        style: after.style,
-        child: MentionBlock(
-          key: ValueKey(mention.mentionId),
-          userRoomId: roomId,
-          node: node,
-          index: index,
-          mentionAttributes: mention,
-        ),
-      );
-    }
-
-    return defaultTextSpanDecoratorForAttribute(
+    TextSpan fallback() => defaultTextSpanDecoratorForAttribute(
       context,
       node,
       index,
       text,
       before,
       after,
     );
+
+    final roomId = widget.roomId;
+
+    final href = attributes[AppFlowyRichTextKeys.href] as String?;
```

Comment:
for href we could  use ```defaultTextSpanDecoratorForAttribute``` instead, this inbuilt function handles href attribute also